mimic/test/test_identity_objects.py

```diff
@@ -0,0 +1,302 @@
+from __future__ import absolute_import, division, unicode_literals
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from mimic.model.identity_objects import (
+    bad_request,
+    not_found,
+    forbidden
+)
+from mimic.test.dummy import (
+    ExampleEndPointTemplate,
+    make_example_external_api
+)
+
+
+class YetToBeDone(SynchronousTestCase):
+    """
+    Test that are a placeholder for necessary functionality
+    that will be needed when the full functionality is implemented.
+    """
+
+    def test_stuff_todo(self):
+        """
+        Temporary for code-coverage until end-points are
+        implemented that will actually use these.
+        """
+        class reqMock(object):
+            def setResponseCode(self, code):
+                pass
+
+        bad_request("testing bad request", reqMock())
+        not_found("testing not found", reqMock())
+        forbidden("testing forbidden", reqMock())
```

Comment:
This is a couple things that I know I'll need in the next PR and had added. I'd prefer to keep them in this PR and promise they will be used in the next one or removed. In fact, I'll likely be removing `forbidden` in the next PR, but `bad_request` and `not_found` are both utilized; and `YetToBeDone` will be removed in the PR.

The test is there just to ensure sufficient code coverage instead of adding them and letting them just hang out. As noted, it's just temporary.